README.md

```diff
@@ -63,11 +61,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Vet Clinic] <a name="about-project"></a>
```

Comment:
- [ ] Kindly add a project description, this will make your readme more descriptive 👍🏿

Reply:
👍